### DISCUSSION TITLE
Separate hyperlink targets in MyST?

### DISCUSSION THREAD

**Discussion by astrojuanlu** [executablebooks/meta#549] [Q&A] [answered]
In reST, if I want to separate the link from its target (https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#hyperlink-targets), I write something like

```
This is `a link`_.

.. _a link: https://readthedocs.org
```

How would I do that in MyST? I tried locating it in https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#targets-and-cross-referencing but I didn't see an example of it.

**Comment by astrojuanlu** [comment, marked as answer]
Self-answer:

```
A [cookiecutter] template...

[cookiecutter]: https://github.com/audreyr/cookiecutter/
```

**Reply by choldgraf** [reply]
Did you find this in the docs somewhere that wasn't obvious? And if so how can we make it more obvious?

**Reply by astrojuanlu** [reply]
Didn't find it in the docs, but rather it appeared after https://github.com/executablebooks/rst-to-myst 'ing a .rst document, and then I understood. Would be cool to add it to https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#targets-and-cross-referencing
